common/components/common/projects/AboutProjectDisplay.jsx

```diff
@@ -160,6 +162,8 @@ class AboutProjectDisplay extends React.PureComponent<Props, State> {
 
             </React.Fragment>
           }
+          
+          {/*TODO: Groups section*/}
```

Comment:
Do we need this comment now that groups exist?

Reply:
Yes, this is a placeholder for the group entries that will exist on the sidebar.